### PR TITLE
Fixed Issue#582: Prettier Autocomplete

### LINKED ISSUE #582: Better Autocomplete
Idea:
autocomplete

### PR DESCRIPTION
**Pull request by TanSA05**
Steps to solve this issue:
* Added `attribute`s for number of guides and `crop_image` in `crop_serializer.rb`
* Added a custom template for the `typeahead` directive in Angular in this file
* Added CSS and styling for the autocomplete to match the design given by @roryaronson 

This PR fixes #582

### CODE REVIEW

File: app/assets/stylesheets/styles/ui/inputs/typeahead_search.scss [hunk 1]
```diff
@@ -9,7 +9,11 @@
       max-width: 100%;
 
       &.active {
-        background-color: lighten($of-green, 40%);
+        background-color: $of-green;
+
+        a {
+          color: #ffffff;
```
Comment (TanSA05) [inline app/assets/stylesheets/styles/ui/inputs/typeahead_search.scss]: I'm not exactly sure of what to replace it with, @simonv3

Comment (TanSA05) [inline app/assets/stylesheets/styles/ui/inputs/typeahead_search.scss]: I'm not exactly sure of what to replace it with, @simonv3

Comment (simonv3) [inline app/assets/stylesheets/styles/ui/inputs/typeahead_search.scss]: @TanSA05 Have a look at the variables defined in `_variables.scss`

Comment (TanSA05) [inline app/assets/stylesheets/styles/ui/inputs/typeahead_search.scss]: Changed it to `$white` now. Thanks, @simonv3!

**Comment by simonv3** [comment]
Dunno what's up with coveralls. Will merge!

**Comment by simonv3** [comment]
Thanks so much @TanSA05, this great. I'll deploy it right now :).

**Pull request merged**